src/app/admin/store/reducers/admin-books.reducer.ts

```diff
@@ -0,0 +1,16 @@
+import { createReducer, on } from '@ngrx/store';
+
+import * as AdminPageActions from '../actions/admin-books.actions';
+import { initialAdminBookState } from '../state/admin-books.state';
+
+export const adminBooksReducers = createReducer (
+  initialAdminBookState,
+  on(AdminPageActions.BookAddedSuccess,
+    (state, { selectedBook }) => ({ ...state, selectedBook })),
+  on(AdminPageActions.BookAddedError,
+    (state) => ({ ...state })),
```

Comment:
I'm not sure it's needed
will try to discuss during call

Reply:
fixed